scripts/bountiful.ash

```diff
@@ -73,7 +73,8 @@ int[item] BAN_ITEMS = {
 // TODO: add other skills (mostly IOTMs I don't have)
 boolean[skill] BAN_SKILLS = {
   $skill[Snokebomb] : true,           // Snojo
-  $skill[Talk About Politics] : true  // Pantsgiving
+  $skill[Talk About Politics] : true,  // Pantsgiving
+  $skill[Bowl a Curveball] : true
```

Comment:
do we want this to be more substantial? feels like a very limited list we could add to if we really want to

Reply:
Ya totally agree - plan to include more in a future PR. I'll make an issue to track this when that section is enabled

Reply:
Created issue #3